pyproject.toml

```diff
@@ -53,7 +53,7 @@ testing-core = [
     "pytest-cov==6.0.0; python_version>='3.9'",
 ]
 testing = [
-    "tripper[testing-core,mappings,datadoc]",
+    "tripper[datadoc,mappings,testing-core]",
```

Comment:
Var ikke klar over at rekkefølgen har noe å si. Eller er det bare for å få det i alfabetisk rekkefølge?